src/components/add-tag-dialog/add-tag-dialog.js

```diff
@@ -34,6 +31,21 @@ export class AddTagComponent {
     this.dialog.listen("MDCDialog:cancel", this.handleCancel.bind(this));
   }
 
+  clean() {
+    this.tagNameInput.value = "";
+  }
+
+  handleOk() {
+    this.props.onAddTagConfirm({
+      name: this.tagNameInput.value
+    });
+    this.clean();
+  }
+
+  handleCancel() {
+    console.log("declined");
```

Comment:
you need this handle to clean your dialog. it is no need in `console.log` anymore